zephyr/esp32s3/include/bt/esp_bt.h

```diff
@@ -304,6 +326,14 @@ typedef void (* btdm_vnd_ol_task_func_t)(void *param);
     .ble_data_lenth_zero_aux = BT_BLE_ADV_DATA_LENGTH_ZERO_AUX,            \
     .ble_chan_ass_en = BT_CTRL_CHAN_ASS_EN,                                \
     .ble_ping_en = BT_CTRL_LE_PING_EN,                                     \
+    .ble_llcp_disc_flag = 0,                                               \
```

Comment:
Keep hardcode for now. Will submit an exclusive PR to include necessary kconfigs. The way it is set, it used default values.